utils/logger.js

```diff
@@ -38,35 +38,21 @@ function formatMessage(message, meta) {
   return message;
 }
 
-module.exports.log =
-  (message, meta) => {
-    if (logdnaLogger) {
-      logdnaLogger.log(formatMessage(message, meta), { meta });
+function commonLogger(type) {
+  return (message, meta) => {
+    if (type !== undefined) {
+      if (logdnaLogger) {
+        logdnaLogger[type](formatMessage(message, meta), { meta });
+      }
+      bunyanLogger[type] = (message, meta || '');
```

Comment:
That won't work, you're setting `bunnyanLogger` instead of executing it, change to: `bunyanLogger[type](message, meta || '');`

Reply:
@bluzi  i tried that already.But its throwing a error

```
/home/r-ganesh/github/travis-buddy/utils/logger.js:47
      bunyanLogger[type](message, meta || '');
                        ^

TypeError: bunyanLogger[type] is not a function
```

Reply:
Right, that makes sense, since, while logDNA have `log`, bunyan doesn't and it has `debug` instead. 

IMHO, you should refactor `commonLogger` so it will take 2 parameters instead of 1, the first will be `logdnaType` and the second will be `bunyanType`, then sue these parameters as the type of the functions (`logdnaLogger[logdnaType](..)` and `bunyanLogger[bunyanType](...)`) and change the declarations to be something like: `[commonLogger('log', 'debug'), commonLogger('warn'), commonLogger('error'), commonLogger('debug')]`

This way, each logger will use its function.
Hope it's clear.

Let me know if you disagree